### PR TITLE
l10n: uk: update core translation

### PR DESCRIPTION
**Pull request by kasya**

### CODE REVIEW

File: po/uk.po [hunk 1]
```diff
@@ -2154,81 +2143,81 @@ msgstr ""
...
 
 #: wt-status.c:1254
 msgid "  (use \"git am --skip\" to skip this patch)"
-msgstr ""
+msgstr "  (використовуйте команду \"git am --skip\", щоб пропустити цей патч)"
 
 #: wt-status.c:1257
 msgid "  (use \"git am --allow-empty\" to record this patch as an empty commit)"
-msgstr ""
+msgstr "  (використовуйте \"git am --allow-empty\", щоб записати цей патч як порожній комміт)"
```
Comment (arkid15r) [inline po/uk.po]:
```suggestion
msgstr "  (використовуйте \"git am --allow-empty\", щоб записати цей патч як порожній коміт)"
```

File: po/uk.po [hunk 2]
```diff
@@ -2154,81 +2143,81 @@ msgstr ""
...
 
 #: wt-status.c:1259
 msgid "  (use \"git am --abort\" to restore the original branch)"
-msgstr ""
+msgstr "  (для відновлення початкової гілки використовуйте \"git am --abort\")"
```
Comment (arkid15r) [inline po/uk.po]:
```suggestion
msgstr "  (використовуйте \"git am --abort\", щоб відновити початкову гілку)"
```

File: po/uk.po [hunk 3]
```diff
@@ -2154,81 +2143,81 @@ msgstr ""
...
 
 #: wt-status.c:1392
 msgid "git-rebase-todo is missing."
-msgstr ""
+msgstr "git-rebase-todo відсутній."
 
 #: wt-status.c:1394
 msgid "No commands done."
-msgstr ""
+msgstr "Ніяких команд не виконано."
```
Comment (arkid15r) [inline po/uk.po]:
```suggestion
msgstr "Не виконано жодної команди."
```

File: po/uk.po [hunk 4]
```diff
@@ -2154,81 +2143,81 @@ msgstr ""
...
 #: wt-status.c:1397
 #, c-format
 msgid "Last command done (%<PRIuMAX> command done):"
 msgid_plural "Last commands done (%<PRIuMAX> commands done):"
-msgstr[0] ""
-msgstr[1] ""
-msgstr[2] ""
+msgstr[0] "Останню команду виконано (%<PRIuMAX> команду виконано):"
+msgstr[1] "Останні команди виконано (%<PRIuMAX> команди виконано):"
+msgstr[2] "Останні команди виконано (%<PRIuMAX> команд виконано):"
```
Comment (arkid15r) [inline po/uk.po]:
```suggestion
msgstr[2] "Останніх команд виконано (%<PRIuMAX> команд виконано):"
```

File: po/uk.po [hunk 5]
```diff
@@ -2154,81 +2143,81 @@ msgstr ""
...
 
 #: wt-status.c:1408
 #, c-format
 msgid "  (see more in file %s)"
-msgstr ""
+msgstr "  (див. більше у файлі %s)"
```
Comment (arkid15r) [inline po/uk.po]:
```suggestion
msgstr "  (дивіться більше у файлі %s)"
```

File: po/uk.po [hunk 6]
```diff
@@ -2154,81 +2143,81 @@ msgstr ""
...
+msgstr "Не залишилось команд."
 
 #: wt-status.c:1416
 #, c-format
 msgid "Next command to do (%<PRIuMAX> remaining command):"
 msgid_plural "Next commands to do (%<PRIuMAX> remaining commands):"
-msgstr[0] ""
-msgstr[1] ""
-msgstr[2] ""
+msgstr[0] "Наступна команда для виконання (залишилась %<PRIuMAX> команд):"
```
Comment (arkid15r) [inline po/uk.po]:
```suggestion
msgstr[0] "Наступна команда для виконання (залишилась %<PRIuMAX> команда):"
```

File: po/uk.po [hunk 7]
```diff
@@ -2154,81 +2143,81 @@ msgstr ""
...
 #: wt-status.c:1424
 msgid "  (use \"git rebase --edit-todo\" to view and edit)"
-msgstr ""
+msgstr "  (використовуйте \"git rebase --edit-todo\" для перегляду та редагування)"
 
 #: wt-status.c:1436
 #, c-format
 msgid "You are currently rebasing branch '%s' on '%s'."
-msgstr ""
+msgstr "Наразі ви перебазовуєте гілку ‘%s’ на ‘%s’."
```
Comment (arkid15r) [inline po/uk.po]:
```suggestion
msgstr "Наразі ви перебазовуєте гілку '%s' на '%s'."
```

File: po/uk.po [hunk 8]
```diff
@@ -2293,8 +2281,7 @@ msgstr "  (скористайтесь командою \"git cherry-pick —-ski
 
 #: wt-status.c:1522
 msgid "  (use \"git cherry-pick --abort\" to cancel the cherry-pick operation)"
-msgstr ""
-"скористайтесь командою \"git cherry-pick -—abort\", щоб скасувати операцію висмикування"
+msgstr "скористайтесь командою \"git cherry-pick -—abort\", щоб скасувати операцію висмикування"
```
Comment (arkid15r) [inline po/uk.po]:
```suggestion
msgstr " (скористайтесь командою \"git cherry-pick -—abort\", щоб скасувати операцію висмикування)"
```

File: po/uk.po (hunk 6 above)
Comment (kasya) [reply po/uk.po]: 💯

File: po/uk.po (hunk 7 above)
Comment (kasya) [reply po/uk.po]: Good catch!

**Pull request merged**